pkg/support/dump_test.go

```diff
@@ -38,3 +46,38 @@ func TestParseFileName(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, ts.String(), "2021-08-17 09:47:58 +0000 UTC")
 }
+
+func TestTimestampFilter(t *testing.T) {
+	since := "7s"
+	ts := timestampFilter(since)
+	currentTime := time.Now()
+	assert.Contains(t, ts.String(), currentTime.Format("2006-01-02"))
+}
+
+func TestWriteFile(t *testing.T) {
+	fs := afero.NewOsFs()
+	resource := "networkpolicies"
+	basedir := "dir1/dir2"
+	data := []byte("antct -oyaml get networkpolicies")
+	err := writeFile(fs, filepath.Join(basedir, resource), resource, data)
+	if err != nil {
+		require.EqualError(t, err, "error when writing networkpolicies to file: open dir1/dir2/networkpolicies: no such file or directory")
```

Comment:
here you have to be sure with the test case that whether the error will be generated or not, and you can cover these in two separate test cases. Also when you are checking for no error you should also check the contents of the file written.